data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+		payment 600000
+		conversation
+			`Coming in for a landing, you receive a message from the the Heliarch elder Iekie.`
+			`	"Captain <last>! Analyzed your findings, we have. A monetary bonus, sending, we are, as thanks."`
+			`	<payment> are transferred to your account.`
+				decline
+
+
+
+mission "HI2-Bonus 3"
+	invisible
+	landing
+	to offer
+		has "Heliarch Investigation 2: done"
+		"resistance evidence" == 3
+	source
+		government "Coalition" "Heliarch"
+	on offer
+		payment 900000
+		conversation
+			`Coming in for a landing, you receive a message from the the Heliarch elder Iekie.`
+			`	"Captain <last>! Analyzed your findings, we have. A monetary bonus, sending, we are, as thanks."`
+			`	<payment> are transferred to your account.`
+				decline
+
+
+
+mission "HI2-MP"
+	name "Investigate Mebla's Portion"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Mebla's Portion"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`You arrive to a thunderstorm, as is many times the norm for this planet.`
+			`	Tugroob suggests that he pose as an Arach writer looking to find housing on the planet, and scurries off to what must be a real estate office. Iplora continues with the original idea of pretending to be a tourist, saying that she will get by asking the young local Arachi, hoping to find something of use.`
+			`	You are left with Kirrie, who has been mostly silent as his friends left your ship.`
+			choice
+				`	"Are you not going to follow their lead?"`
+				`	"Have you not decided how to investigate the planet?"`
+			`	"Captain <last>. Give us an edge, your human approach to these missions would, is what the elder said. Only, underused, your foreign ideas are, if only the order at which we land on these planets, they are used for."`
+			`	He looks out to the rainy spaceport, some rain drops are starting to get inside your ship, since the hatch is still open.`
+			`	"Lead the way, would you? The warehouses, I wish to inspect, but your assistance I could use."`
+			choice
+				`	"Of course, I'll get my rain coat."`
+					goto wetbeetle
+				`	"I'd rather not go out there with that rain."`
+					goto lonebeetle
+			label wetbeetle
+			`	You walk in a straight line for about an hour, many times without any building providing cover from the rain. Kirrie takes you to the very edge of the town; you would be able to see the hills and mountains in the distance, were it not for the colossal rain clouds and the nonstop precipitation blocking your view. One or two warehouses have a bit of rust on their doors, and rarely you spot one with a damaged roof, some rain creeping inside. You can also make out the figures of small trees and bushes though, not too far out.`
```

Comment:
`	You walk in a straight line for about an hour, many times without any building providing cover from the rain. Kirrie takes you to the very edge of the town where you would be able to see the hills and mountains in the distance, were it not for the colossal rain clouds and the nonstop precipitation blocking your view. One or two warehouses have a fair bit of rust on their doors, and just once did you spot one with a damaged roof. You can also make out the shapes of small trees and bushes not too far out.`